### ISSUE TITLE
Custom Tile Overlay not working properly

### ISSUE THREAD

**Issue by Stophface** [react-community/react-native-maps#1815]
Hey guys, 
I am just wondering if anyone of you has some ideas regarding this:

I am using `<MapView.UrlTile .../>` to add a `Custom Tile Overlay`.

```
<MapView 
   ...>
  <MapView.UrlTile
    urlTemplate={this.state.urlTemplate}
  />
</MapView>
```

 It sort of works.

When I pinch, drag and zoom in/out, it displays the underlaying tiles (on Android Google Maps). Once the pinching, dragging, zooming is done, it only then loads the `Tiles` passed with `<MapView.UrlTile .../>`. It looks like basically it loads two maps. First the underlaying, and then the one passed through `<MapView.UrlTile .../>`.
Here a Video: [https://www.file-upload.net/download-12828114/maps_prob1.mov.html](https://www.file-upload.net/download-12828114/maps_prob1.mov.html) (19.41 MB, download link)

**Comment by compojoom** [comment]
@Stophface - that's the way custom tiles normally work. You have your base map below and on top of that you have the custom tile. It's actually supposed that the custom tile is semi transparent - like rain, clouds, wind etc. In your case you seem to be loading a completely custom map. Can't you use the custom maps for this?

**Comment by Stophface** [comment]
@compojoom I am using OpenStreetMap `http://c.tile.openstreetmap.org/{z}/{x}/{y}.png` Where is the difference between a `Custom Map` and a `Custom Tile Overlay` when I want the OpenStreetMap Map as my displayed map?

**Comment by alvelig** [comment]
Duplicate of #773 then
